packages/sui-test-contract/package.json

```diff
@@ -17,8 +17,8 @@
   },
   "dependencies": {
     "@pact-foundation/pact": "9.18.1",
-    "@pactflow/pact-msw-adapter": "1.2.1",
-    "@s-ui/mock": "1",
+    "@pactflow/pact-msw-adapter": "1.5.0",
+    "@s-ui/mock": "beta",
```

Comment:
We'll change it when the new `sui-mock` version is released in the next master workflow execution.